lib/OpenQA/Utils.pm

```diff
@@ -558,5 +560,19 @@ sub job_groups_and_parents {
     return \@res;
 }
 
+sub wait_with_progress {
+    my $interval = ceil(shift);
```

Comment:
well, better or worse is subjective - but at least it's how we do it elsewhere :)